modules/rosa-hcp-sts-creating-a-cluster-cli.adoc

```diff
@@ -28,35 +28,44 @@ When using the {product-title} (ROSA) CLI, `rosa`, to create a cluster, you can
 //REGION="<region>"
 //----
 
-. You can create your {hcp-title} cluster with one of the following commands.
+. The following command creates your {hcp-title} cluster.
 +
 [NOTE]
 ====
 When creating a {hcp-title} cluster, the default machine Classless Inter-Domain Routing (CIDR) is `10.0.0.0/16`. If this does not correspond to the CIDR range for your VPC subnets, add `--machine-cidr <address_block>` to the following commands. To learn more about the default CIDR ranges for {product-title}, see xref:../networking/cidr-range-definitions.adoc#cidr-range-definitions[CIDR range definitions].
 ====
 +
-** Create a cluster with a single, initial machine pool, publicly available API, and publicly available Ingress by running the following command:
-+
+--
 [source,terminal]
 ----
-$ rosa create cluster --cluster-name=<cluster_name> \
-    --sts --mode=auto --hosted-cp --operator-roles-prefix <operator-role-prefix> \
-    --oidc-config-id <ID-of-OIDC-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id>
+$ rosa create cluster --cluster-name=<cluster_name> \ <.>
+    --sts --mode=auto --hosted-cp [--private] \ <.> 
+    --operator-roles-prefix <operator-role-prefix> \ <.>
+    --oidc-config-id <id-of-oidc-configuration> --subnet-ids=<public-subnet-id>,<private-subnet-id>
 ----
+<.> Specify the name of your cluster, ensuring that the name is not longer than 15 characters.
```

Comment:
I'm curious why mention the limit of 15 characters alone. To be comprehensive, you must say what are the rules for cluster -name. For example:

> Cluster name must consist of no more than 15 lowercase alphanumeric characters or '-', start with a letter, and end with an alphanumeric character.